server/src/main/java/org/kaazing/gateway/server/config/parse/translate/June2016Validator.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kaazing.gateway.server.config.parse.translate;
+
+import org.kaazing.gateway.server.config.parse.translate.june2016.RemoveRequireUser;
+
+public class June2016Validator extends GatewayConfigTranslatorPipeline {
+
+    public June2016Validator() {
+        super();
+        
+        // Set the namespace
+        addTranslator(new RemoveRequireUser());   
```

Comment:
We should be removing the realm on the old namespace only if there is a realm and no authorization constraint.  Meaning the translator is wrong and this should be added to a different Translator Pipeline